### ISSUE TITLE
About the weights for cross entropy loss

### ISSUE THREAD

**Issue by dzzhang96** [yhygao/CBIM-Medical-Image-Segmentation#15]
Hi yhygao, thank you for the codes. I have seen you have used `weight: [0.5, 1, 3]  # weight of each class in the loss function` when training the LITS datasets. I was now using my own datasets to segment the lung tumor. Can I ask for your experience if the weights are appropriate if I set it to [0.5, 2] or [0.5, 3]? Thanks again

**Comment by yhygao** [comment]
Hi,
It depends on how imbalanced your dataset is. The model is more likely to predict it to be a tumor instead of a background if the weight is high. But false positives may happen. In my experience, this weight doesn't affect a lot if your training epoch is long enough. Both [0.5, 2] and [0.5, 3] should be ok. I can't say which is better, you'd better try it by yourself. I recommend you to visualize the segmentation maps to see how many are the false positive/negatives.

**Comment by dzzhang96** [comment]
Thank you!
